### PR TITLE
docs upate v4: readme & part docs develop;

### PR DESCRIPTION
**Pull request by xia-huang-411303**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
+&nbsp;
 ## What is EasyRec?
+---
 
 ![intro.png](docs/images/intro.png)
 
 ### EasyRec is an easy to use framework for Recommendation
 
-EasyRec致力于成为容易上手的工业界深度学习推荐算法框架，支持大规模训练、评估、导出和部署。EasyRec实现了业界领先的模型，包含排序、召回、多目标等模型，支持超参搜索，显著降低了建模的复杂度和工作量。
+EasyRec implements state of the art machine learning models used in common recommedation tasks: candidate generation(matching), scoring(ranking), and multi-task learning. It improves the efficiency of generating high performance models by simple configuration and hyper parameter tuning(HPO).
```
Comment (poson) [inline README.md:11]: machine learning models -> deep learning models

File: README.md [hunk 2]
```diff
@@ -44,31 +49,58 @@ EasyRec致力于成为容易上手的工业界深度学习推荐算法框架，
...
-```
+&nbsp;
+## Contact 
+---
 
-- [EasyRec Framework](https://easyrec.oss-cn-beijing.aliyuncs.com/docs/EasyRec.pptx)
+### Join Us
 
-- [Run](docs/source/quick_start/local_tutorial.md)
+- DingDing Group: 32260796. (Easy_Rec usage general discussion.)
```
Comment (poson) [inline README.md:96]: Easy_Rec  ->EasyRec

File: docs/source/develop.md [hunk 3]
```diff
@@ -1,85 +1,90 @@
...
 pre-commit run -a
 ```
 
-### Test
+### 测试
+
+#### 单元测试
 
-#### Unit test
+TEST_DEVICES=0,1 sh scripts/ci_test.sh
```
Comment (poson) [inline docs/source/develop.md:43]: 重复了

File: README.md [hunk 4]
```diff
@@ -44,31 +49,58 @@ EasyRec致力于成为容易上手的工业界深度学习推荐算法框架，
...
 
-- [PAI-DSW DEMO](https://dsw-dev.data.aliyun.com/#/?fileUrl=http://easyrec.oss-cn-beijing.aliyuncs.com/dsw/easy_rec_demo.ipynb&fileName=EasyRec_DeepFM.ipynb)
-  (Remember to select Python 3 kernel)
+- Email Group: easy_rec@service.aliyun.com.
 
-- [Develop](docs/source/develop.md)
+### Enterprise Service
+- If you need Easy_Rec enterprise service support, or purchase cloud product services, you can contact us by DingDing Group.
```
Comment (poson) [inline README.md:101]: Easy_Rec  -> EasyRec

**Review by dawn310826 — APPROVED** [review]

**Pull request merged**